README.md

```diff
@@ -1,22 +1,52 @@
 # VirtoCommerce.Cart
-VirtoCommerce.Cart module represents shopping cart management system. This module doesn't have any UI in VC Manager. 
 
-# Documentation
-User guide:
+## Overview
 
-Developer guide:
+VirtoCommerce.Cart module represents shopping cart management system. This module doesn't have any UI in VC Manager.
 
-# Installation
+Shopping cart module manages customers accumulated list of items, calculates a total for the order, including shipping and handling (i.e., postage and packing) charges and the associated taxes.
+
+![Cart Module](docs/media/screen-cart-module.png)
+
+![Cart Module Info](docs/media/screen-cart-module-info.png)
+
+## Features
+
+1. Supports multiple carts
+1. Wishlist
+1. Grouping multiple carts to one order
+1. Anonymous carts
+1. Stock reservation
+1. Multiple payments methods
+1. Create new cart from orders history
+
+The main purpose of the cart module is to save and store the shopping cart data and changes on VC Manager side.
+
+The cart module is connected with the shopping cart via two types of API requests:
+
+1. General API request that allows saving the entire shopping cart information on the database;
+1. Specific API requests that allow call specific operations, for example delete, edit, choose payment method, choose delivery type, etc.
+
+For more details about the available cart-module API, please follow the link bellow:
+
+https://admin-demo.virtocommerce.com/docs/ui/index#/Shopping%2520cart%2520module
+
+The cart-module functionality can be extended and customized based on the specific business needs. In order to extend the cart module functionality, use the carte module template that can be accessed by following the link below:
```

Comment:
change
"use the carte module template" to
"use "Virto Commerce 2.x Cart and Order Module extension" template"